README.md

```diff
@@ -3,336 +3,91 @@
+
+An option writer's claim to the underlying asset(s) (if not exercised) and exercise asset(s) (if exercised) is represented by a non-fungible Option Lot Claim token. This Claim NFT can be redeemed for their share of the underlying plus exercise assets, based on currently exercised.
+
+The structure of an option is as follows:
+
+- **Underlying asset:** the ERC-20 address of the asset to be received
+- **Underlying amount:** the amount of the underlying asset contained within an option of this type
+- **Exercise asset:** the ERC-20 address of the asset needed for exercise
+- **Exercise amount:** the amount of the exercise asset required to exercise this option
+- **Earliest exercise timestamp:** the timestamp after which this option can be exercised
+- **Expiry timestamp:** the timestamp before which this option can be exercised
+
+The Core is unopinionated on the type of option (call vs. put), where, when, or for how much an option is bought/sold, and whether or not the option is profitable when exercised. Because all options written with Valorem are fully collateralized, physical settlement at exercise or redeem time is instant and gas-efficient.
+
+Read the [litepaper](https://valorem.xyz/docs/valorem-options-litepaper/) for more insight into the business use cases that Valorem enables.
+
+## Trust Model
+### Actors
+There are 3 main actors in the core protocol:
+- Protocol Admin
+- Option Writer
+- Option Holder
+
+The **Protocol Admin** is the address to which protocol fees are swept, and can update the Protocol Admin address. No other permissioned actions are possible.
+
+**Option Writers** can create new option types and write option lots for any valid ERC-20 asset pair (excluding rebasing, fee-on-transfer, and ERC-777 tokens). Sufficient approval must be granted for the Settlement Engine to take custody of the requisite amount of the underlying asset. Once an option lot expires, the writer can redeem their Claim NFT to their share of the underlying and/or exercise assets.
+
+**Option Holders** can buy options that have been written. This is accomplished via a standard ERC-1155 transfer of the desired amount of option contracts from writer to holder. When exercising an option, they must have enough of the exercise asset, and similarly to when writing an option, they must have granted sufficient approval for the Settlement Engine on the ERC20 exercise asset.
+
+### Assets
+When an option is written, the Settlement Engine takes custody of the underlying asset, transferring in the requisite amount. When an option is exercised, the Engine transfers in the requisite amount of the exerise asset and transfers out the underlying asset. When an option lot claim is redeemed, the Engine transfers out that claimant's share of the exercise asset and possible unexercised underlying asset.
+
+### Actions
```

Comment:
This should be removed from the readme to a separate `actions.md`

Reply:
Nah, I don't think it is, it's not targeting the reader, it's targeting a protocol auditor/tester. Different audience.

Reply:
Okay pulling out